### PR TITLE
Release 5.1.0

### PR DESCRIPTION
**Pull request by kitchoi**
Preparing for the next release.

- [x] Add tox
- [x] Run tox with PySide on Python 2.7 and 3.4
- [x] Run tox with wxPython on Python 2.7 **Error: same error as on Travis CI*
- [ ] Run tox with wxPython on Python 3.4

### CODE REVIEW

**Comment by corranwebster** [comment]
Can we get #277 into this release, please?

**Comment by kitchoi** [comment]
@corranwebster  Done.

**Comment by kitchoi** [comment]
@corranwebster, @jdmarch  If no more PR needs merging, I am going to flag the IS_RELEASE and tag in the next few hours.

**Comment by kitchoi** [comment]
@corranwebster Ready for merging

**Comment by corranwebster** [comment]
Assuming tests pass, LGTM

File: travis-ci-requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+nose
```
Comment (itziakos) [inline travis-ci-requirements.txt]: nose is already installed in travis-ci

**Comment by kitchoi** [comment]
Second to last commit has passed all tests.  Last commit only concerns installing already-installed nose on travis, whose tests have passed.  Appveyor is taking too long to queue.  Merging now.

**Pull request merged**